Src/Support/GoogleApis.Auth.DotNet4.Tests/Program.cs

```diff
@@ -0,0 +1,41 @@
+// ***********************************************************************
```

Comment:
Eh. NUnitLite puts this file in the project on install and the contents are pretty trivial -- I'm not really worried about the duplication.